src/erlcloud_cloudwatch_logs.erl

```diff
@@ -437,6 +464,80 @@ log_stream_order_by(undefined) -> <<"LogStreamName">>;
 log_stream_order_by(log_stream_name) -> <<"LogStreamName">>;
 log_stream_order_by(last_event_time) -> <<"LastEventTime">>.
 
+%%------------------------------------------------------------------------------
+%% @doc
+%%
+%% GetQueryResults action
+%% https://docs.aws.amazon.com/AmazonCloudWatchLogs/latest/APIReference/API_GetQueryResults.html
+%%
+%% ===Example===
+%%
+%%   Returns the results from the specified query.
+%%
+%% `
+%%   application:ensure_all_started(erlcloud).
+%%   {ok, Config} = erlcloud_aws:auto_config().
+%%   {ok, Results} = erlcloud_cloudwatch_logs:get_query_results("12ab3456-12ab-123a-789e-1234567890ab").
```

Comment:
Would it be better to pass `Config` here explicitly?

Reply:
Sure: 87868d5.